app/src/main/java/com/android/voyageur/ui/search/SearchScreen.kt

```diff
@@ -24,8 +50,148 @@ fun SearchScreen(
             selectedItem = navigationActions.currentRoute())
       },
       content = { pd ->
-        Text(
-            modifier = Modifier.padding(pd).testTag("emptySearchPrompt"),
-            text = "You aren't able to search for anything yet.")
+        Column(
+            modifier =
+                Modifier.padding(pd)
+                    .fillMaxSize()
+                    .background(Color.White)
+                    .testTag("searchScreenContent")) {
+              Spacer(modifier = Modifier.height(24.dp))
+              Text(
+                  text = "Search",
+                  color = Color.Black,
+                  style = MaterialTheme.typography.bodyLarge,
+                  fontSize = 24.sp,
+                  modifier = Modifier.padding(start = 16.dp, bottom = 8.dp))
+
+              Row(
+                  modifier =
+                      Modifier.padding(horizontal = 16.dp)
+                          .fillMaxWidth()
+                          .background(Color.LightGray, shape = MaterialTheme.shapes.medium)
+                          .padding(8.dp)
+                          .testTag("searchBar"),
+                  verticalAlignment = Alignment.CenterVertically) {
+                    Icon(Icons.Default.Search, contentDescription = null)
+                    Spacer(modifier = Modifier.width(8.dp))
+                    BasicTextField(
+                        value = searchQuery,
+                        onValueChange = {
+                          searchQuery = it
+                          userViewModel.searchUsers(searchQuery.text)
+                        },
+                        modifier = Modifier.weight(1f).padding(8.dp).testTag("searchTextField"),
+                        textStyle = LocalTextStyle.current.copy(fontSize = 18.sp),
+                    )
+                    Icon(
+                        Icons.Default.List,
+                        contentDescription = null,
```

Comment:
I noticed that the contentDescription for icons and images is currently set to null. Providing meaningful contentDescription values for all icons and images is important for accessibility. Implementing these descriptions can significantly improve the accessibility of the app. It ensures that assistive technologies can accurately convey the purpose of each visual element.